.vscode/settings.json

```diff
@@ -4,6 +4,5 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": "always"
   },
-  "eslint.validate": ["javascript", "typescript"],
-  "cSpell.language": "en,fr"
+  "eslint.validate": ["javascript", "javascriptreact", "typescript", "typescriptreact"]
```

Comment:
This was causing the issue in my editor. I work with my editor opened at the monorepo root, not each individual project.